extensions/vscode/src/views/homeView.ts

```diff
@@ -780,13 +784,26 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     this.requestWebviewSaveSelection();
   }
 
-  private async showSelectOrCreateConfigForDeployment() {
+  private getCredentialNameForContentRecord(
```

Comment:
Perhaps we should change this to return the entire `Credential | undefined` then do `?.name` on the result just to make it more general purpose.